tap_five9/__init__.py

```diff
@@ -141,7 +141,7 @@ def do_sync(client, catalog, state, start_date):
                 client=client,
                 start_date=start_date
             )
-        counter_value = sync_stream(state, start_date, instance)
+        counter_value = sync_stream(state, client.config, start_date, instance)
```

Comment:
We don't need to pass in `client.config` anymore since the `ReportStream.sync` method only accepts `state` as an arg.

Reply:
hmm although this is definitely true, needs to be fixed

Reply:
ah jk im dumb, didn't have all the streams selected (face palm). seeing the error locally when I run now, thanks for catching!
![Screen Shot 2023-01-24 at 9 13 18 AM](https://user-images.githubusercontent.com/29642644/214361496-5d81109a-8ce0-4f64-b708-bdcf802927ad.png)